modern-business/style-editor.scss

```diff
@@ -27,9 +28,7 @@ Modern Business Editor Styles
 		$main-block-selector: ".wp-block[data-align=\"#{$alignment}\"]";
 	}
 
-	@return "
-	  #{$main-block-selector},
-	  .wp-block[data-type=\"a8c/post-content\"] #{$main-block-selector}";
+	@return '#{$main-block-selector}, .wp-block[data-type=\"a8c/post-content\"] #{$main-block-selector}';
```

Comment:
Will need updated once https://github.com/Automattic/themes/pull/1257 is in

Reply:
all we need to do is change `.wp-block[data-type=\"a8c/post-content\"]` to `.wp-block.post-content__block` and rebuild the CSS and the conflicts should be fine 👍

Reply:
Yeah, this change was just to clear the linter, it's not related to the PR.
It's also the only conflict so it's a super easy fix! 🙂